pre-registration/pre-registration-application-service/src/main/java/io/mosip/preregistration/application/service/LoginService.java

```diff
@@ -423,4 +437,21 @@ public void setAuditValues(String eventId, String eventName, String eventType, S
 		res.setResponsetime(GenericUtil.getCurrentResponseTime());
 		return res;
 	}
+
+	public String generateJWTToken(String userId, String issuerUrl) {
+		log.info("sessionId", "idType", "id", "In generateJWTToken method of loginservice:" + userId + issuerUrl);
+		Map<String, Object> claims = new HashMap<String, Object>();
+		claims.put("userId", userId);
+		claims.put("scope", PreRegLoginConstant.JWT_SCOPE);
+		claims.put("user_name", userId);
+		claims.put("roles", jwtTokenRoles);
+
+		String jws = Jwts.builder().setClaims(claims).setIssuer(issuerUrl).setIssuedAt(Date.from(Instant.now()))
```

Comment:
This should be explicitly set to UTC, Our server could run in a different timeszone and the user could be accessing it from a different timezone. As the expiry can be read both in client and server its preferred to have this in UTC.